release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -715,19 +720,27 @@ The following related APIs have also been removed:
 * `.osb.openshift.io/v1`
 
 Service Catalog and Template Service Broker will be removed in a future
-{product-title} release. If they are enabled in 4.4, the web console warns the
-user that these features are still enabled.
+{product-title} release, as well as the following related API:
 
-The following alerts can be viewed from the *Monitoring* -> *Alerting* page and
-have a *Warning* severity:
+* `.servicecatalog.k8s.io/v1beta1`
+
+If they are enabled in 4.4, the web console warns cluster administrators that
+these features are still enabled. The following alerts can be viewed from the
+*Monitoring* -> *Alerting* page and have a *Warning* severity:
 
 * `ServiceCatalogAPIServerEnabled`
 * `ServiceCatalogControllerManagerEnabled`
 * `TemplateServiceBrokerEnabled`
 
-The following related API will be removed in a future release:
+The `service-catalog-controller-manager` and `service-catalog-apiserver` cluster
+Operators are also now set to `Upgradeable=false` in 4.4, meaning they will
+block future cluster upgrades to the next minor version, for example 4.5, if
+they are still installed at that time. Upgrading to z-stream releases such as
+4.4.z, however, are still permitted in this state.
 
-* `.servicecatalog.k8s.io/v1beta1`
+If Service Catalog is installed, cluster administrators can see
+xref:../../applications/service_brokers/uninstalling-service-catalog.adoc#sb-uninstalling-service-catalog[Uninstalling Service Catalog]
+to uninstall it before the next minor version of {product-title} is released.
```

Comment:
@shawn-hurley @jmrodri I've updated this to remove the procedure and instead link to a new "Uninstalling Service Catalog" topic. The open PR for that against `master` branch is https://github.com/openshift/openshift-docs/pull/21577, but it looks like:

http://file.rdu.redhat.com/~adellape/042720/uninstall-svccat/applications/service_brokers/uninstalling-service-catalog.html